src/theme/Theme.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from 'styled-components';
+
+const theme = {
+  colors: {
+    black: '#000',
+    white: '#fff',
+    greyLight: '#e6e6e6',
+    greyDark: '#636363',
+    buttonBg: '#fdb755',
+  },
+  textsize: {
+    big: '38px',
+    body: '16px',
+    small: '14px',
+    smaller: '13px',
+  },
+  pageContainer: {
+    position: 'relative',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    minHeight: '100vh',
+    padding: '122px 0 100px 0',
```

Comment:
I guess it would be better to have `padding: 100px 0;` here since that's the height of header and footer. But we can adjust that if necessary on the Search page as well